extensions/kubectl-cli/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "kubectl-cli-tool",
+  "displayName": "kubectl CLI",
+  "description": "Install and update kubectl CLI Tools without leaving Podman Desktop",
+  "version": "0.0.1",
+  "icon": "icon.png",
+  "publisher": "redhat",
+  "license": "Apache-2.0",
+  "engines": {
+    "podman-desktop": "^0.0.1"
+  },
+  "main": "./dist/extension.js",
+  "scripts": {
+    "build": "vite build && node ./scripts/build.js",
+    "test": "vitest run --coverage",
+    "watch": "vite build -w",
+    "format:check": "prettier --check \"**/*.ts\" \"scripts/*.js\"",
+    "format:fix": "prettier --write \"**/*.ts\" \"scripts/*.js\""
```

Comment:
We already have the format scripts in the main package.json. Any reason to have it here as well?

Reply:
It just works much faster when it checks only extension.